src/@next/components/molecules/CardHeader/CardHeader.tsx

```diff
@@ -24,9 +23,7 @@ export const CardHeader: React.FC<IProps> = ({
   return (
     <S.Header divider={divider}>
       <Text {...getTitleProps(textStyle, titleSize)}>{children}</Text>
-      {closeIcon && (
-        <S.CloseBtn onClick={onHide}>{renderCloseIcon()}</S.CloseBtn>
-      )}
+      {onHide && <S.CloseBtn onClick={onHide}>{renderCloseIcon()}</S.CloseBtn>}
       {customIcon && <S.CloseBtn>{customIcon}</S.CloseBtn>}
```

Comment:
As discussed in private: condition and CloseBtn wrapper

Reply:
fixed, and Icon replaced with IconButton